services/infra/InfraService.ts

```diff
@@ -11,41 +12,87 @@ interface InfraServiceProps {
   sitesService: SitesService
   reposService: ReposService
   deploymentsService: DeploymentsService
+  mailer: MailClient
 }
 
 export default class InfraService {
-  private readonly usersService: InfraServiceProps["usersService"]
+  private readonly usersService
 
-  private readonly sitesService: InfraServiceProps["sitesService"]
+  private readonly sitesService
 
-  private readonly reposService: InfraServiceProps["reposService"]
+  private readonly reposService
 
-  private readonly deploymentsService: InfraServiceProps["deploymentsService"]
+  private readonly deploymentsService
+
+  private readonly mailer
 
   constructor({
     usersService,
     sitesService,
     reposService,
     deploymentsService,
+    mailer,
   }: InfraServiceProps) {
     this.usersService = usersService
     this.sitesService = sitesService
     this.reposService = reposService
     this.deploymentsService = deploymentsService
+    this.mailer = mailer
+  }
+
+  sendCreateError = async (
+    email: string,
+    repoName: string | undefined,
+    submissionId: string | undefined,
+    error: string
+  ) => {
+    const subject = `[Isomer] Create site ${repoName} FAILURE`
+    const html = `<p>Isomer site ${repoName} was <b>not</b> created successfully. (Form submission id [${submissionId}])</p>
+<p>${error}</p>
+<p>This email was sent from the Isomer CMS backend.</p>`
+    await this.mailer.sendMail(email, subject, html)
+  }
+
+  sendCreateSuccess = async (
+    email: string,
+    repoName: string,
+    submissionId: string | undefined,
```

Comment:
could i double check why `submissionId` could be `undefined`?

Reply:
This is no longer necessary.